README.md

```diff
@@ -25,6 +25,7 @@ latest (9.0) : https://download.checkmarx.com/9.0.0/Plugins/CxConsolePlugin-2020
 2020.2.7 : https://download.checkmarx.com/9.0.0/Plugins/CxConsolePlugin-2020.2.7.zip
 
 2020.2.3 : https://download.checkmarx.com/9.0.0/Plugins/CxConsolePlugin-2020.2.3.zip
+<<<<<<< HEAD
```

Comment:
@nleach999 seems like the merge didn't went ok

Reply:
ok, I think I got it right this time